utils/src/main/java/uk/gov/di/authentication/utils/lambda/EmailCheckResultWriterHandler.java

```diff
@@ -59,7 +68,11 @@ public Void emailCheckResultWriterHandler(SQSEvent event) {
                 LOG.info(
                         "Message for email check reference {} written to database",
                         emailCheckResult.referenceNumber());
+                var timeFromInitialRequest =
+                        now().getTime() - emailCheckResult.timeOfInitialRequest();
 
+                cloudwatchMetricsService.putEmbeddedValue(
+                        "NumberOfAccounts", timeFromInitialRequest, Map.of());
```

Comment:
Should this be something like `EmailCheckDuration` rather than `NumberOfAccounts`?

Reply:
Yes, thanks!